applications/HDF5Application/CMakeLists.txt

```diff
@@ -5,50 +5,44 @@ message("**** configuring KratosHDF5Application ****")
 include_directories( ${KRATOS_SOURCE_DIR}/kratos )
 
 # generate variables with the sources
-set( KRATOS_HDF5_APPLICATION_SOURCES
+set( KRATOS_HDF5_APPLICATION_CORE
 	${CMAKE_CURRENT_SOURCE_DIR}/hdf5_application.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/hdf5_application_variables.cpp
+  	${CMAKE_CURRENT_SOURCE_DIR}/hdf5_application_variables.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_file.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_file_serial.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_model_part_io.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_properties_io.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_data_value_container_io.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_nodal_solution_step_variables_io.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_nodal_solution_step_data_io.cpp
-  ${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_nodal_solution_step_bossak_io.cpp
+  	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_nodal_solution_step_bossak_io.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_points_data.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_connectivities_data.cpp
 	${CMAKE_CURRENT_SOURCE_DIR}/custom_io/hdf5_container_component_io.cpp
```

Comment:
`GLOB` could make your life easier here...

Reply:
I was thinking about it, but was not so confident because if the ordering is messed up, then it might complain or not :/ (specialliy in the case of templates, I experiencecd similar case in StatApp)